python-backend/app/api/documents/expected/resources/documents.py

```diff
@@ -13,31 +13,47 @@
 
 class ExpectedDocumentResource(Resource, UserMixin, ErrorMixin):
     parser = reqparse.RequestParser()
-    parser.add_argument('document', type=dict, required=True,
-                        help='document to change', location="json")
+    parser.add_argument(
+        'document',
+        type=dict,
+        required=True,
+        help='document to change',
+        location="json")
 
-    @api.doc(params={'exp_doc_guid': 'Required: Mine number or guid. returns list of expected documents for the mine'})
+    @api.doc(
+        params={
+            'exp_doc_guid':
+            'Required: Mine number or guid. returns list of expected documents for the mine'
+        })
     @jwt.requires_roles(["mds-mine-view"])
     def get(self, exp_doc_guid=None):
         if exp_doc_guid is None:
-            return self.create_error_payload(404, 'Must provide a expected document guid.'), 404
+            return self.create_error_payload(
+                404, 'Must provide a expected document guid.'), 404
         mine_exp_doc = ExpectedDocument.find_by_exp_document_guid(exp_doc_guid)
         if mine_exp_doc is None:
-            return self.create_error_payload(404, 'Expected document not found'), 404
+            return self.create_error_payload(
+                404, 'Expected document not found'), 404
         return {'expected_document': mine_exp_doc.json()}
 
-    @api.doc(params={'exp_doc_guid': 'Required: Mine number or guid. Updates expected document'})
+    @api.doc(params={
+        'exp_doc_guid':
+        'Required: Mine number or guid. Updates expected document'
+    })
     @jwt.requires_roles(["mds-mine-create"])
     def put(self, exp_doc_guid=None):
         if exp_doc_guid is None:
-            return self.create_error_payload(404, 'Must provide a expected document guid.'), 404
+            return self.create_error_payload(
+                404, 'Must provide a expected document guid.'), 404
 
         exp_doc = ExpectedDocument.find_by_exp_document_guid(exp_doc_guid)
         if exp_doc is not None:
             data = self.parser.parse_args()
             updated_doc = data['document']
-            if str(exp_doc.exp_document_guid) != updated_doc['exp_document_guid']:
-                return self.create_error_payload(500, 'exp_document does not match guid provided'), 500
+            if str(exp_doc.
```

Comment:
What's determining the location of these line breaks?

Reply:
That would be the Python formatter YAPF and I don't understand why it would choose that as its break point.

Reply:
pep8 standard is max line length of 80, YAPF and BLACK both enforce 85-90 as max line length as it breaks significantly fewer lines.

Reply:
YAPF by default it 80, which is fairly strict. I'd look at changing that a bit, but if that's the case we should version the settings file so everyone has the same. Discussion for another time.

Reply:
we had already had "PEP8 80 character limit is stoopid" discussion, with @samfresh  and @mmumby  @ConradGustafson  at the start.   Our other projects have more reasonable line limits set ... GWELLS is at least 110... if the MDS dev team agrees, we can shift back to "more than 80"